packages/core/types/Config.tsx

```diff
@@ -38,15 +38,18 @@ export type ComponentConfig<
     params: {
       changed: Partial<Record<keyof FieldProps, boolean>>;
       lastData: DataShape | null;
+      metadata: MetaData;
     }
   ) =>
     | Promise<{
         props?: Partial<FieldProps>;
         readOnly?: Partial<Record<keyof FieldProps, boolean>>;
+        renderProps?: Partial<FieldProps>;
       }>
     | {
         props?: Partial<FieldProps>;
         readOnly?: Partial<Record<keyof FieldProps, boolean>>;
+        renderProps?: Partial<FieldProps>;
```

Comment:
What are these `renderProps` type changes for?

Reply:
Removed. Artifact from a previous implementation attempt.